packages/react-native/ReactCommon/react/renderer/graphics/Transform.h

```diff
@@ -101,7 +101,8 @@ struct Transform {
    */
   static Transform FromTransformOperation(
       TransformOperation transformOperation,
-      const Size& size);
+      const Size& size,
+      const Transform& transform = Transform::Identity());
```

Comment:
@javache Should we need to keep `transform` optional to make API stable?

Reply:
This seems ok to me. As long as C++ doesn't do anything silly like promoting all callers to value copies instead of refs.